department-of-veterans-affairs/abd-vro
# Max CFI: Add support for other diagnostic codes

## What was the problem?
Currently, only the max rating for tinnitus (dc 6260) is available. This PR adds the remaining diagnostic codes for over 700 other rated disabilities. 

Associated tickets or Slack threads:
- resolves #2748 

## How does this fix it?[^1]
Adds support for other diagnostic codes

## How to test this PR
- unit tests

[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
